src/components/Input.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState} from 'react';
+import Button from './Button';
+
+
+const Input=()=> {
+    const [value, setValue] = useState<string>("");
+
+    function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
+        setValue(event.currentTarget.value);
```

Comment:
You should trim the string before setValue handle cases where the input is just a space (' ').

Reply:
Thank you didn't think of this case. Have made the change now for the case where input is just one or more whitespaces.